WeakAuras/Types.lua

```diff
@@ -198,7 +198,8 @@ WeakAuras.race_types = {
   HighmountainTauren = LBR("Highmountain Tauren"),
   Nightborne = LBR("Nightborne"),
   DarkIronDwarf = LBR("Dark Iron Dwarf"),
-  ZandalariTroll = LBR("Zandalari Troll")
+  ZandalariTroll = LBR("Zandalari Troll"),
+  MagharOrc = LBR("Mag'har Orc"),
```

Comment:
Mmm, this isn't really supposed to be in the PR really. But I suppose it would've needed to be added eventually anyways. Note that as of today, it's a no-op since LBR doesn't have maghar orc data.